stubs/fpdf2/fpdf/fpdf.pyi

```diff
@@ -240,7 +257,10 @@ class FPDF:
     def get_y(self) -> float: ...
     def set_y(self, y: float) -> None: ...
     def set_xy(self, x: float, y: float) -> None: ...
-    def output(self, name: str = ..., dest: str = ...): ...
+    @overload
+    def output(self, name: Literal[""] = ...) -> bytearray: ...
+    @overload
+    def output(self, name: str) -> None: ...
```

Comment:
Strictly speaking, the second overload should be `Not[Literal[""]]`. If a `str` argument is provided, it could be the empty string and therefore would return `bytearray`. But I believe that this is a fringe case and it's safer to just return `None` in that case, instead of `Any`.